### ISSUE TITLE
Install inside the docker: photoneo theme folder is missing in your working directory

### ISSUE THREAD

**Issue by blackball** [photoneo/phoxi_camera#35]
I am using a ubuntu 16.04 docker to install the latest photoneo ubuntu SDK.

After installation, and I ran it. It gives me an runtime exception error:

```
theme folder is missing in your working directory 
```

Can you guys have a quickly test ? Providing a docker image will be very helpful since the SDK is only supporting ubuntu up to 16.04, which makes working with photoneo cameras really a pain nowadays. 

Thanks.

**Comment by lihk11** [comment]
I got the same problem, but in ubuntu 18.04 instead of a docker

**Comment by blackball** [comment]
> I got the same problem, but in ubuntu 18.04 instead of a docker... Did you solve it?

No, I didn't fix the problem for Photoneo, but made my own 3D camera with friendly SDK across Windows, Linux, Android.

**Comment by lihk11** [comment]
> No, I didn't fix the problem for Photoneo, but made my own 3D camera with friendly SDK across Windows, Linux, Android.
Wow!  awesome! ! 
When I saw "friendly SDK across Windows, Linux, Android", I knew it's RVBust. And it turns out you are the boss.
After tried your camera，I have to say,  your SDK are the most convenient one i ever used. It only took me 10 minutes for getting the first demo up. 
Extraordinary work!

**Comment by blackball** [comment]
@lihk11 What a small world! Thanks for your comment on our camera. It is getting better day by day. Despite the cross-platform SDK, we have added lots of new algorithms, like noise removal algorithm. Also we will open source our Python binding codes in near future. ;-)

**Comment by marcelsvec** [comment]
@lihk11 @blackball I'm sorry I removed your comments as they were not relevant to the topic.

**Comment by Abeljin** [comment]
I got the same problem，how to use photoneo in a ubuntu 16.04 docker

**Comment by Abeljin** [comment]
PhoXi Control 1.2.14 - Commit ID: #7b10b08
/root/.PhotoneoPhoXiControl
terminate called after throwing an instance of 'std::runtime_error'
  what():  theme folder is missing in your working directory

**Comment by Abeljin** [comment]
> > PhoXi Control 1.2.14 - Commit ID: #7b10b08
> > /root/.PhotoneoPhoXiControl
> > terminate called after throwing an instance of 'std::runtime_error'
> > what(): theme folder is missing in your working directory
> 
> export PHOXI_CONTROL_PATH=
> 
> PhoXi Control 1.2.14 - Commit ID: #7b10b08
> /root/.PhotoneoPhoXiControl
> Developer
> PhoXiControl: browser.c:581: avahi_service_browser_new: Assertion `client' failed.
> Aborted (core dumped)

qt5 dockerfile
https://github.com/theshadowx/Qt5_Docker/blob/default_qt5_xenial/Dockerfile

**Comment by zhubeilife** [comment]
hi @Abeljin @blackball @lihk11 have you guy fix this one? I have the same issue. 
Sorry for the bothering

**Comment by martin4951** [comment]
Same issue here.
